cscs-checks/tools/profiling_and_debugging/mpip_mpi_omp.py

```diff
@@ -0,0 +1,95 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.required_version('>=2.14')
+@rfm.parameterized_test(['Cpp'], ['F90'])
+class MpipCheck(rfm.RegressionTest):
+    '''This test checks mpiP, the light-weight MPI profiler:
+       http://llnl.github.io/mpiP
+    '''
+    def __init__(self, lang):
+        super().__init__()
+        self.valid_systems = ['daint:gpu', 'daint:mc',
+                              'dom:gpu', 'dom:mc']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        # -g compilation flag is needed to report source code filename and line
+        self.prgenv_flags = {
+            'PrgEnv-cray': ['-g', '-h nomessage=3140', '-homp', '-O2'],
+            'PrgEnv-gnu': ['-g', '-fopenmp', '-O2'],
+            'PrgEnv-intel': ['-g', '-qopenmp', '-O2'],
+            'PrgEnv-pgi': ['-g', '-mp', '-O2']
+        }
+        self.modules = ['mpiP']
+        self.build_system = 'Make'
+        self.iterations = 500
```

Comment:
Why not set the name of this variable from `self.iterations` to `self.num_iterations`?

Reply:
👍